### PR TITLE
Bug 1551711 - Added alt-text prop to all snippet schemas that include an icon.

### PR DESCRIPTION
**Pull request by AllegroFox**
Fallback is an empty string.

### CODE REVIEW

File: content-src/asrouter/templates/SimpleBelowSearchSnippet/SimpleBelowSearchSnippet.schema.json [hunk 1]
```diff
@@ -29,6 +29,11 @@
       "type": "string",
       "description": "Snippet icon. Dark theme variant. 64x64px. SVG or PNG preferred."
     },
+    "icon_alt_text": {
+      "type": "string",
+      "description": "Alt text for accessibility",
```
Comment (k88hudson) [inline content-src/asrouter/templates/SimpleBelowSearchSnippet/SimpleBelowSearchSnippet.schema.json]: Maybe "Alt text describing icon for screen readers"

Comment (AllegroFox) [reply content-src/asrouter/templates/SimpleBelowSearchSnippet/SimpleBelowSearchSnippet.schema.json]: that sounds perfect, I'll do that

**Comment by AllegroFox** [comment]
Discussed with @glogiotatidis to make sure this PR is compatible with Snippets work; got the all clear :)

**Comment by glogiotatidis** [comment]
> Discussed with @glogiotatidis to make sure this PR is compatible with Snippets work; got the all clear :)

ftr I checked Schemas, chosen variable names are OK. Also that we have a default value in case the server doesn't not send alt text. 

lgtm! :cake:

File: content-src/asrouter/templates/SimpleBelowSearchSnippet/SimpleBelowSearchSnippet.jsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import {safeURI} from "../../template-utils";
 import {SnippetBase} from "../../components/SnippetBase/SnippetBase";
 
 const DEFAULT_ICON_PATH = "chrome://branding/content/icon64.png";
-// Alt text if available; in the future this should come from the server. See bug 1551711
+// Alt text placeholder in case the prop from the server isn't available; See bug 1551711
```
Comment (k88hudson) [inline content-src/asrouter/templates/SimpleBelowSearchSnippet/SimpleBelowSearchSnippet.jsx]: Can you remove the bug reference

**Pull request merged**